core-web/libs/portlets/edit-ema/portlet/src/lib/dot-ema-shell/store/dot-ema.store.ts

```diff
@@ -19,14 +19,19 @@ import {
 import { DEFAULT_PERSONA } from '../../shared/consts';
 import { EDITOR_STATE } from '../../shared/enums';
 import { ActionPayload, SavePagePayload } from '../../shared/models';
-import { insertContentletInContainer, sanitizeURL } from '../../utils';
+import {
+    createIframeUrlByRenderedPage,
+    insertContentletInContainer,
+    sanitizeURL
+} from '../../utils';
 
 export interface EditEmaState {
     clientHost: string;
     error?: number;
     editor: DotPageApiResponse;
     isEnterpriseLicense: boolean;
     editorState: EDITOR_STATE;
+    vtlIframeContent?: string;
```

Comment:
Why do we need this here, if you are adding it to the selector in line 91.

Reply:
Initially I wanted a variable to know if it was a vtl or headless, but you right, with the clientHost we can know it
Removed!